proto/engine/v1/execution_engine.proto

```diff
@@ -125,3 +125,9 @@ message ForkchoiceState {
 	bytes safe_block_hash      = 2 [(ethereum.eth.ext.ssz_size) = "32"];
 	bytes finalized_block_hash = 3 [(ethereum.eth.ext.ssz_size) = "32"];
 }
+
+message Withdrawal {
+	uint64 index = 1;
+	bytes address = 2 [(ethereum.eth.ext.ssz_size) = "20"];
+	uint64 amount = 3;
+}
```

Comment:
Withdrawal is already defined here: https://github.com/prysmaticlabs/prysm/blob/78c55019e629885622f565c3e074e141d6b9f50e/proto/prysm/v1alpha1/withdrawals.proto#L28

Reply:
I think we should move withdrawal from prysm/proto/prysm/v1alpha1/withdrawals.proto to proto/engine/v1/withdrawals.proto to live next to implementation. Any opposition to this?